lib/devise.rb

```diff
@@ -153,7 +153,11 @@ module Test
   mattr_accessor :pepper
   @@pepper = nil
 
-  # Used to enable sending notification to user when their password is changed
+  # Used to send notification to the original user email when their email is changed.
+  mattr_accessor :send_email_change_notification
```

Comment:
I'd vote for deprecating `password_change` too, but dunno if we need to do it right away or save for a future breaking release.